frontend/lib/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -45,15 +45,17 @@ registerLoaders([CSVLoader, GLTFLoader])
 const EMPTY_LAYERS: LayersList = []
 
 export const DeckGlJsonChart: FC<PropsWithHeight> = props => {
-  const { element, height, isFullScreen, width } = props
+  const { element, isFullScreen } = props
   const theme: EmotionTheme = useTheme()
-  const { createTooltip, deck, onViewStateChange, viewState } = useDeckGl({
-    element,
-    isLightTheme: hasLightBackgroundColor(theme),
-    width,
-    height,
-    isFullScreen,
-  })
+
+  const { createTooltip, deck, onViewStateChange, viewState, width, height } =
+    useDeckGl({
+      element,
+      isLightTheme: hasLightBackgroundColor(theme),
+      width: props.width,
+      height: props.height,
```

Comment:
super nit (personal preference): I personally would destructure `props.width` and `props.height` [above](https://github.com/streamlit/streamlit/pull/9464/files#diff-75e7f4652df3884dab3b3edaa2507d4393363d902403af9ac95889bd0ce88bc4R48) and rename it there to `propsWidt`, `propsHeight` so that we have one place where we see what props are used and we don't mix different styles of accessing.